### ISSUE TITLE
Add select to input link as options to: http://, mailto:, whatsapp://

### ISSUE THREAD

**Issue by lpirola** [nossas/slate-editor#29]

**Comment by gabrielrtakeda** [comment]
@lpirola, I was thinking to avoid the behaviour to add the **http://** automatically when, the user want to link to another route of the same URL.. Like in this mobilization: https://www.metasdesp.minhasampa.org.br/ where these 4 buttons on the first block, points to the same URL with a hash.. Instead of user insert the whole URL like https://www.metasdesp.minhasampa.org.br/#block-4610 on the link's href, he can link it simply using the hash **#block-4610**.. After this explanation, what do you think, it is really a good idea to force the **http://** protocol?

**Comment by lpirola** [comment]
@gabrielrtakeda no, looks fine for me. Maybe we could add a message, warning user about that?

**Comment by gabrielrtakeda** [comment]
@lpirola what do you think about some message like this:
<img width="799" alt="screen shot 2017-06-05 at 12 40 15" src="https://cloud.githubusercontent.com/assets/5435389/26791032/3d7bc058-49ec-11e7-92d7-90c80b6afcc0.png">
